### ISSUE TITLE
Use setup.cfg for python distribution

### ISSUE THREAD

**Issue by mh-northlander** [WorksApplications/sudachi.rs#116]
Static metadata is now recommended.
ref: https://packaging.python.org/tutorials/packaging-projects/#configuring-metadata

**Comment by eiennohito** [comment]
I don't think that we can do that with Rust. It actually executes non-default Python code to call cargo/rust toolchain to compile Rust module.

**Comment by eiennohito** [comment]
Also, see https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html, but I don't think that we should move anywhere from setup.py-based installation for the time being.

**Comment by mh-northlander** [comment]
We still need to use `setup.py`, but I feel `setup.cfg` is handy for some metadata like version or readme.

**Comment by eiennohito** [comment]
Then we also probably should add `pypackage.toml` with setuptools-rust/wheel dev dependencies written there.
